test/alphabet/alphabet_test.cpp

```diff
@@ -49,14 +49,16 @@ using alphabet_types = ::testing::Types<dna4, dna5, rna4, rna5, nucl16,
                                         union_composition<dna4>,
                                         union_composition<dna4, gap>,
                                         union_composition<dna4, dna5, gap>,
+                                        char, char16_t, // char32_t, too slow
+                                        uint8_t, uint16_t, // uint32_t, too slow
```

Comment:
What means to slow? Shouldn't that be documented somewhere, that we discourage using `char32_t`, if it is to slow?